File: benchmarks/python/mandelbrot.py
inside = 0
for py in range(400):
    y = (py / 400.0) * 2.0 - 1.0
    for px in range(400):
        x = (px / 400.0) * 3.0 - 2.0
        zx = zy = 0.0
        i = 0
        while i < 50 and zx * zx + zy * zy <= 4.0:
            zx, zy = zx * zx - zy * zy + x, 2.0 * zx * zy + y
            i += 1
        if i == 50:
            inside += 1
print(inside)

File: benchmarks/node/mandelbrot.js
let inside = 0;
for (let py = 0; py < 400; py++) {
  const y = (py / 400) * 2 - 1;
  for (let px = 0; px < 400; px++) {
    const x = (px / 400) * 3 - 2;
    let zx = 0, zy = 0, i = 0;
    while (i < 50 && zx * zx + zy * zy <= 4) {
      const t = zx * zx - zy * zy + x;
      zy = 2 * zx * zy + y;
      zx = t;
      i++;
    }
    if (i === 50) inside++;
  }
}
console.log(inside);
